src/brush.rs

```diff
@@ -37,7 +48,8 @@ impl From<Image> for Brush {
 
 impl Default for Brush {
     fn default() -> Self {
-        Self::Solid(Color::default())
+        // FIXME(color): Decide what to do about color and defaults
+        Self::Solid(color::palette::css::TRANSPARENT)
```

Comment:
Current Peniko defaults to transparent. Opaque black can be considered: it is visible, and has a natural representation in the color spaces we include in `color`. The other special color is white, though the physical color it represents depends on the color space.

Reply:
There was a previous PR with discussion about it: https://github.com/linebender/peniko/pull/31 and that links to more discussion in Xilem.

We'll also have to decide whether to solve it here in the `Brush::default()` only or also in `color` or both.

Reply:
Also, with https://github.com/linebender/color/issues/63 in place, this would be shorter …

Reply:
I don't think we should solve this in Color.